components/brave_shields/browser/cookie_list_opt_in_service.h

```diff
@@ -14,14 +14,17 @@
 #include "mojo/public/cpp/bindings/receiver_set.h"
 #include "mojo/public/cpp/bindings/remote_set.h"
 
+class PrefService;
+
 namespace brave_shields {
 class AdBlockService;
 
 // This class is not thread-safe and should have single owner
 class CookieListOptInService : public KeyedService,
                                public mojom::CookieListOptInPageAndroidHandler {
  public:
-  explicit CookieListOptInService(AdBlockService* ad_block_service);
+  explicit CookieListOptInService(AdBlockService* ad_block_service,
```

Comment:
remove `explicit`

Reply:
why?

Reply:
`Constructors that cannot be called with a single argument may omit explicit. Constructors that take a single std::initializer_list parameter should also omit explicit, in order to support copy-initialization (e.g., MyType m = {1, 2};).`

https://google.github.io/styleguide/cppguide.html#Implicit_Conversions